drizzlepac/devutils/plot_ci_vs_fwhm.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+import argparse
+import pdb
+
+from astropy.stats import sigma_clipped_stats
+from astropy.table import Table, vstack
+import matplotlib.pyplot as plt
+import numpy as np
+from drizzlepac.devutils.comparison_tools import compare_sourcelists
+
+"""
+1: read in ci_fwhm file(s)
+2: smash all input datasets into a single dataset
+3: determine min, max CI values
+4: set up CI value bins based on dataset min/max values and specified bin size
+5: for each bin, compute resistant mean FWHM value, sigma value
+6: plot bin CI value vs. mean FWHM values
+"""
+# -~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-
+def get_data(input_files):
+    """read in data from one or more files. In the case that there are multiple files, the data from all files
+    will be combined into a single CI vs FWHM dataset.
+
+    Parameters
+    ----------
+    input_files : list
+        list of CI vs. FWHM .csv files to process
+
+    Returns
+    -------
+    data_table: : numpy.ndarray
+        A 2 x n sized numpy array. Column 1: CI values; Column 2: FWHM values
+    """
+    data_table = Table()
+    for filename in input_files:
+        data_table = vstack([data_table,Table.read(filename,format='ascii.csv')])
+    return data_table
+
+# -~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-
+
+def run_stuff(input_files,bin_size,ci_limits,plot_title):
```

Comment:
It may be a little less casual to call this something like 'run_all' or even just simply 'run' instead.  Not a required change, but it did stand out.

Reply:
Resolved by commit [b936ca3](https://github.com/spacetelescope/drizzlepac/pull/602/commits/b936ca39eb52f04864bd2c71741d3f29040fa99a)